packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -734,7 +734,7 @@ const DatePickerInput = (props) => {
       pattern="\d*"
       aria-required={required}
       aria-label={intl.formatMessage({ id: 'Terra.datePicker.dayLabel' })}
-      aria-describedby={dateFormatOrder === DateUtil.dateOrder.DMY ? `${nameLabelId} ${ariaDescriptionIds}` : ariaDescriptionIds}
+      aria-describedby={`${nameLabelId} ${ariaDescriptionIds}`}
```

Comment:
```suggestion
      aria-describedby={`${nameLabelId}, ${ariaDescriptionIds}`}
```

Reply:
The utilization of commas in this context may lead to suboptimal behavior with assistive technologies -as per the MDN websites